CMakeLists.txt

```diff
@@ -354,10 +354,6 @@ target_include_directories(
     zeek_dynamic_plugin_base SYSTEM
     INTERFACE $<INSTALL_INTERFACE:include/zeek/3rdparty/prometheus-cpp/include>)
 
-target_include_directories(
-    zeek_dynamic_plugin_base SYSTEM
-    INTERFACE $<INSTALL_INTERFACE:include/zeek/3rdparty/expected-lite/include>)
-
```

Comment:
The commit title is misleading if you wanted to fix that up. We continue to install it (and need to as the files are included from util.h), the change here removes adding a non-existing directory to dynamic plugin's target include_directories which failed to compile.